brh.data-commons.org/metadata/aggregate_config.json

```diff
@@ -1,135 +1,285 @@
-      "field_mappings": {
-        "authz": "path:authz",
-        "tags": "path:gen3_discovery.tags",
-        "_unique_id": "path:_unique_id",
-        "study_id": "path:_unique_id",
-        "study_description": "path:description",
-        "full_name": "path:full_name",
-        "short_name": "path:full_name",
-        "commons": "Kids First Data Resource Center",
-        "study_url": "path:link",
-        "__manifest": [],
-        "commons_url": "kidsfirstdrc.org"
-      }
-    }
-  }
+	"configuration": {
+		"schema": {
+			"_subjects_count": {
+				"type": "integer"
+			},
+			"__manifest": {
+				"type": "array",
+				"properties": {
+					"file_name": {
+						"type": "string"
+					},
+					"file_size": {
+						"type": "integer"
+					}
+				}
+			},
+			"tags": {
+				"type": "array"
+			},
+			"_unique_id": {},
+			"study_description": {},
+			"study_id": {},
+			"study_url": {},
+			"project_id": {},
+			"short_name": {},
+			"full_name": {},
+			"commons_url": {},
+			"commons" : {},
+			"authz": {
+				"type": "string"
+			}
+		},
+		"settings" : {
+			"cache_drs" : true
+		}
+	},
+	"adapter_commons": {
+		"IBD Commons": {
+			"mds_url": "https://ibdgc.datacommons.io/",
+			"commons_url" : "ibdgc.datacommons.io",
+			"adapter": "gen3",
+			"config" : {
+				"guid_type": "my_metadata",
+				"study_field": "my_metadata"
+			},
+			"keep_original_fields": false,
+			"field_mappings" : {
+				"authz": "path:authz",
+				"tags": "path:tags",
+				"_unique_id": "path:_unique_id",
+				"study_id": "path:_unique_id",
+				"study_description": "path:brief_summary",
+				"full_name": "path:dataset_title",
+				"short_name": "path:dataset_code",
+				"commons": "NIDDK IBD Genetics Consortium Data Commons",
+				"study_url": "path:link",
+				"_subjects_count" : "path:subjects_count",
+				"__manifest": "path:__manifest",
+				"commons_url" : "ibdgc.datacommons.io"
+			}
+		},
+		"BioData Catalyst": {
+			"mds_url": "https://gen3.biodatacatalyst.nhlbi.nih.gov/",
+			"commons_url" : "gen3.biodatacatalyst.nhlbi.nih.gov",
+			"adapter": "gen3",
+			"config" : {
+				"guid_type": "discovery_metadata",
+				"study_field": "gen3_discovery"
+			},
+			"keep_original_fields": false,
+			"field_mappings" : {
+				"authz": "path:authz",
+				"tags": "path:tags",
+				"_unique_id": "path:study_id",
+				"study_id": "path:study_id",
+				"study_description": "path:study_description",
+				"full_name": "path:full_name",
+				"short_name": "path:short_name",
+				"commons": "BioData Catalyst",
+				"study_url": "path:dbgap_url",
+				"_subjects_count" : {"path":"_subjects_count", "default" : 0 },
+				"__manifest": "path:__manifest",
+				"commons_url" : "gen3.biodatacatalyst.nhlbi.nih.gov"
+			}
+		},
+		"MIDRC": {
+			"mds_url": "https://data.midrc.org/",
+			"commons_url" : "data.midrc.org",
+			"adapter": "gen3",
+			"config" : {
+				"guid_type": "discovery_metadata",
+				"study_field": "discovery_metadata"
+			},
```

Comment:
ah I see `study_data_field` is changed to `study_field`

Reply:
It's the new format and the fact that this configuration is using the Gen3 metadata adapter instead of the older  "gen3_commons" adapter: which needs to be depreciated